pkgs/development/python-modules/django-extensions/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, django, six }:
+
+buildPythonPackage rec {
+  pname = "django-extensions";
+  version = "1.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "08rd9zswvjb9dixzyd3p3l3hw3wwhqkgyjvid65niybzjl1xdb5h";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [ django six ];
+
+  meta = with stdenv.lib; {
+    description = "A collection of custom extensions for the Django Framework";
+    homepage = https://github.com/django-extensions/django-extensions;
+    licenses = [ licenses.mit ];
+  };
```

Comment:
No, I won't.  I don't think I have enough knowledge on python packaging.

I use django for my own personal project and found this project useful. Try to install it and think it would save time for other if the package is ready-to-use.